src/models/contactadmin.js

```diff
@@ -0,0 +1,32 @@
+const {
+    Model
+  } = require('sequelize');
+  module.exports = (sequelize, DataTypes) => {
+    class ContactAdmin extends Model {
+      /**
+       * Helper method for defining associations.
+       * This method is not a part of Sequelize lifecycle.
+       * The `models/index` file will call this method automatically.
+       */
+      static associate(models) {
+        // define association here
+      }
+    }
+    ContactAdmin.init({
+      senderId: DataTypes.INTEGER,
+      asigneeId: DataTypes.INTEGER,
+      email: DataTypes.STRING,
+      messageID: DataTypes.INTEGER,
```

Comment:
Field name needs to be in camelCase. Can we regenerate this model with correct field name?

Reply:
Fixed